### PR TITLE
Add resubmit all to resubmit a list of jobs

### PR DESCRIPTION
**Pull request by mboussaa**
Add the possibility to resubmit a list of jobs when the user selects more than one job. 
The submit window does not show up when we select more than one job.

### CODE REVIEW

**Review by fviale — APPROVED** [review]
Well done !

**Comment by fviale** [comment]
Retest this please

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/RestClient.java [hunk 1]
```diff
@@ -92,6 +92,20 @@ String submitFlat(@HeaderParam("sessionid") String sessionId,
...
+     * @param sessionId
+     *          a valid session id
+     * @param jobsId
+     *          a list of job ids of already submitted jobs
+     * @return a list of job ids of the newly created jobs. If a job submission fails, it will be discarded with a log message.
+     */
+    @POST
+    @Path("jobs/resubmit")
+    @Produces("application/json")
+    InputStream reSubmitAll(@HeaderParam("sessionid") String sessionId, @QueryParam("jobsid") List<String> jobsId);
```
Comment (nebilbenmabrouk) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/RestClient.java:107]: **InputStream** can be converted to **application/json** ?

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java [hunk 2]
```diff
@@ -404,6 +404,35 @@ public void resubmitJob(String jobId) {
...
+    /**
+     * Resubmit a list of jobs
+     *
+     * @param jobIds List of job Ids
+     */
+    public void resubmitAllJobs(List<String> jobIds) {
+        final List<Integer> selectedJobIds = new ArrayList<>(jobIds.size());
+        for (String id : jobIds) {
+            selectedJobIds.add(Integer.parseInt(id));
+        }
```
Comment (nebilbenmabrouk) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java:416]: Don ot understand this instruction: Integer.parseInt(id). id is already an Int !!
Also, rather use **list.stream.map** to apply a transformation to each element in the list without creating a new one.

Comment (fviale) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/JobsController.java:416]: id is a string :-)

**Review by nebilbenmabrouk — APPROVED** [review]

**Comment by nebilbenmabrouk** [comment]
re-test this please

**Pull request merged**